### PR TITLE
Fixing save image

### PR DESCRIPTION
**Pull request by jvarley**
@mateiciocarlie 

This is in response to:
https://github.com/graspit-simulator/graspit/issues/44
While saving an image did not cause an error, it did not actually save an image. 

Changes:
rendered a qimage which allows many different image types to be saved.  

Tested on:
Ubuntu 12.04.

### CODE REVIEW

File: src/ivmgr.cpp [hunk 1]
```diff
@@ -1551,10 +1545,10 @@ IVmgr::saveImage(QString filename)
   renderRoot->addChild(sg);
   
   myRenderer->render(renderRoot);
-  
 
-  myRenderer->writeToFile(SbString(filename.latin1()),
-                          SbName(filename.section('.',-1)));
+  QGLWidget * glWidget = dynamic_cast<QGLWidget *>(renderArea->getGLWidget());
```
Comment (mateiciocarlie) [inline src/ivmgr.cpp]: Should we check for NULL here since it's a dynamic cast?

Comment (jvarley) [inline src/ivmgr.cpp]: done.

**Comment by mateiciocarlie** [comment]
Oh wow, look at that, Travis is online and building this! Great, I guess I'll let it finish :)

**Comment by jvarley** [comment]
I know, I love it.

**Pull request merged**